html/janus.js

```diff
@@ -2269,11 +2269,12 @@ function Janus(gatewayCallbacks) {
 		let groups = {};
 		for(let track of tracks) {
 			delete track.gumGroup;
-			if(!track.type || !['audio', 'video'].includes(track.type))
+			if(!track.type || !['audio', 'video', 'screen'].includes(track.type))
 				continue;
 			if(!track.capture || track.capture instanceof MediaStreamTrack)
 				continue;
-			let group = track.group ? track.group : 'default';
+			// Avoid assigning regular default group if it is a screen capture
+			let group = track.group ? track.group : track.type=='screen'?'sharescreendefault':'default';
```

Comment:
Please put the second block of the ternary operator in brackets to avoid ambiguity, and space things accordingly to make it more readable, e.g.:

    (track.type === 'screen' ? 'sharescreendefault' : 'default')

As a side note, I'm not fond `sharescreendefault` as a name, too long and overly verbose: maybe just `screendefault`? I was going to suggest updating the documentation in `mainpage.dox` accordingly, but I just noticed we don't have any text about `group` in there at all where we talk of `tracks`, so I can add something myself after this is merged.

Reply:
Tweaked code to better fit style as per comments.
Chained ternaries are not really readable but I don't think spinning it off into a function would fit into the current code style.
I'd say `screendefault` sounds good, lets keep that.